### ISSUE TITLE
Deprecation Warnings Unexpected Behaviour

### ISSUE THREAD

**Issue by emmahodcroft** [nextstrain/augur#318]
I went to do a very quick test of the latest `v6` just now, and I'm getting some slightly weird stuff - which seems to be related to the Deprecation warnings ([this commit](https://github.com/nextstrain/augur/commit/9daf76671ac0a4c1586a6cca3c9fc3500920e185)) added by @jameshadfield this morning...?

I had added some new stuff so my whole job started re-running. In `filter`, I got this:
![image](https://user-images.githubusercontent.com/14290674/61458789-9a7d2b80-a96b-11e9-94da-3b025c02b29a.png)

In `refine` I got this:
![image](https://user-images.githubusercontent.com/14290674/61458814-ac5ece80-a96b-11e9-82a3-4cc44e69df7b.png)

I have not been able to replicate this behaviour in `filter` since that first time, but I have in `refine`, multiple times. I modified the `customformatwarning` in `export_v2.py` so that the "DEPRECATED" message was different, and it indeed appears to be what's popping up in `refine`.

If I rewind to the commit just before the Deprecation one, I get neither the Deprecation warning or the "safely interpreted as an integer" warning (in any form).
![image](https://user-images.githubusercontent.com/14290674/61459912-02cd0c80-a96e-11e9-8103-58557e1b3f35.png)

From a little more inspection it seems this happens when numpy (inside TreeTime) tries to send a DeprecationWarning - apparently the changes in warnings in `export_v2.py` effect everything.

```
  File "/home/emma/anaconda3/lib/python3.6/site-packages/treetime/branch_len_interpolator.py", line 39, in __init__
    grid_left = mutation_length * (1 - np.linspace(1, 0.0, n_grid_points/3)**2.0)
  File "/home/emma/anaconda3/lib/python3.6/site-packages/numpy/core/function_base.py", line 101, in linspace
    num = _index_deprecate(num)
  File "/home/emma/anaconda3/lib/python3.6/site-packages/numpy/core/function_base.py", line 21, in _index_deprecate
    warnings.warn(msg, DeprecationWarning, stacklevel=stacklevel)
  File "/home/emma/anaconda3/lib/python3.6/warnings.py", line 101, in _showwarnmsg
    _showwarnmsg_impl(msg)
  File "/home/emma/anaconda3/lib/python3.6/warnings.py", line 28, in _showwarnmsg_impl
    text = _formatwarnmsg(msg)
  File "/home/emma/anaconda3/lib/python3.6/warnings.py", line 116, in _formatwarnmsg
    msg.filename, msg.lineno, line=msg.line)
  File "/mnt/c/Users/Emma/bash/github/augur/augur/export_v2.py", line 35, in customformatwarning
```

This is possibly not the worst thing ever, but it's showing things that usually don't appear (I've never seen either of the above before, they must be below some error display threshold), and also labelling them as 'Deprecated' when they aren't. So we probably need to rein this in a bit!

**Comment by rneher** [comment]
the warnings triggered by treetime should go away after this 

https://github.com/neherlab/treetime/commit/10eb15038844a3f4e803eddaa44b136d5d2088f0

is in the next release.

**Comment by emmahodcroft** [comment]
Definitely not just a treetime problem (though that's good!) The more I play with it, the more weird stuff I get, though. Every time I use `ipdb` I get three Deprecation warnings from somewhere deep in its guts 🙃

**Comment by jameshadfield** [comment]
Ahh - ok, I only tested `augur export` as that was what I was working on, but it seems that it's modified _all_ code uwing the `warnings` module. Will sort out ASAP.

**Comment by jameshadfield** [comment]
Fixed in 33721adb5cd60c9f3411e2189c11c55a1e31591a
